README.md

```diff
@@ -63,6 +63,25 @@ $ wordvecspace convert <input_dir> <output_dir>
 # You can also generate shards by specifying number of vectors per each shard
 $ wordvecspace convert <input_dir> <output_dir> -n 5000
 ```
+
+### wordvecspace Service
```

Comment:
@RamanjaneyuluIdavalapati Just "Service" will do. Also, please move this section and "Interactive console" below "Importing" section. In "Service" section, we need to think about.

1. How does the user know how to make bulk queries?
1. How does the user know what all API methods are available?
1. Do we have to link to `kwikapi` documentation?

Reply:
@prashanthellina 
- We can provide documentation for bulk queries and for all API methods in README.md
- So we don't need to link to kwikapi